File: EtwEvents.Client.Shared/ProcessingOptions.cs
namespace KdSoft.EtwEvents
{
    public class ProcessingOptions
    {
        public int BatchSize { get; set; } = 100;
        public int MaxWriteDelayMSecs { get; set; } = 400;
        public FilterModel Filter { get; set; } = new FilterModel();
    }
}

File: EtwEvents.AgentManager/Spa/js/processingOptions.js
class ProcessingOptions {
  constructor(batchSize, maxWriteDelayMSecs, dynamicParts) {
    this.batchSize = batchSize || 100;
    this.maxWriteDelayMSecs = maxWriteDelayMSecs || 400;
    this.dynamicParts = dynamicParts || [];
  }
}

export default ProcessingOptions;
